client/src/main.cpp

```diff
@@ -3,7 +3,7 @@
 int main()
 {
 	sim::Client client;
-	client.connect_to_server("127.0.0.1", 60000);
+	client.connect_to_server("127.0.0.1", 50000);
```

Comment:
Changed the port from 60000 to 50000 due to a socket connection error during debugging on my current laptop.